src/Components/CollectionPage/BooksCard.js

```diff
@@ -0,0 +1,87 @@
+import React, { useState, useEffect } from 'react';
```

Comment:
Did you make your own version of the Book Card? Or is this Ryan's code showing up here because of merge issues?

Reply:
If you did, I think you can use Ryan's version instead so that we don't have multiple versions of this component!